dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -270,11 +270,19 @@
         IPage<ProcessDefinition> processDefinitionIPage = processDefinitionMapper.queryDefineListPaging(
                 page, searchVal, userId, project.getCode(), isAdmin(loginUser));
 
-        processDefinitionIPage.setRecords(processDefinitionIPage.getRecords());
+        List<ProcessDefinition> records = processDefinitionIPage.getRecords();
+
+        records.forEach(pd -> {
```

Comment:
There is no need to assemble processdefinitionjson for this interface because it cannot be used

Reply:
👌